package.json

```diff
@@ -102,7 +102,7 @@
       },
       {
         "permissionName": "ui-marc-authorities.authority-record.view",
-        "displayName": "View MARC authority record",
+        "displayName": "MARC Authority: View MARC authority record",
```

Comment:
Please also change these in translations

Reply:
Done

Reply:
Done